### DISCUSSION TITLE
[Scam ALert] Artan Life Club شبکه توزیع آرتان

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#30] [Announcements]
<div dir="rtl">

**Project Name:** 
Artan life Club

**نام پروژه:** 

**نماد پروژه:**

**اتفاقی که افتاده است:**

این کمپانی ایرانی است و استارتش از مشهد زده شده
و الان در زاهدان و سبزه وار و... فعالیت دارد

توضیحات یکی از اعضای خودشون:

سلام و درود به طور کلی بخوام براتون توضیح بدم وینرز تیم که یکی از تیمایه قدرتمند بادران بود با تغییر پلن بادران و استخری شدنش تصمیم گرفت جایی شروع به کار کنه که هر چند ماه تغییر پلن نداشته باشه و پورسانت هاش بیشتر باشه و کف مجموعه به هیچ عنوان ضرر نکنه بنابر این جناب گوهر بین راس وینرز تیم با کمک جناب سعادت راس وینرز تیم در مشهد که هر دو خودشون طراح پلن هم هستن پلنی رو ریختن و به چند تا کمپانی که در زمینه رفرال مارکتینگ به صورت حرفه ای داشتند کار میکردن ارائه دادن و گفتن ما به عنوان زیر مجموعه ای مستقل از شما با نام تجاری خودمون و سایت خودمون ولی با خدمات شما حاظر به همکاری هستیم که قوی ترینشون یعنی کمپانی ام وی که ۳ ساله به صورت حرفه ای در زمینه رفرال مارکتینگ در ایران فعالیت داره پذیرفت و ما یعنی کل وینرز تیم که ۸۰۰۰ نفر تقریبا بودیم شیفت شدیم داخل سایت خودمون و هیچ ربطی هم دیگه به بادران نداریم ولی خیلی قدرتمند تریم هم از نظر طرف قرارداد ها و هم از نظر سابقه در رفرال مارکتینگ

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار کلاهبرداری

**نوع کلاهبرداری:** 
طرح پانزی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**
 گوهر بین 
 سعادت

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
https://artanlife.club/
http://www.artannm.ir
https://instagram.com/artan.life?utm_medium=copy_link
https://instagram.com/artanlife_discount?utm_medium=copy_link

**برآورد مبلغ کلاهبرداری:**

**منابع:**
https://www.iranhoshdar.ir/Default.aspx?PageName=Newsb&ID=41499&Language=1

**جمع بندی:**

**Comment by saeedkousha** [comment]
سلام و درود به طور کلی بخوام براتون توضیح بدم وینرز تیم که یکی از تیمایه قدرتمند بادران بود با تغییر پلن بادران و استخری شدنش تصمیم گرفت جایی شروع به کار کنه که هر چند ماه تغییر پلن نداشته باشه و پورسانت هاش بیشتر باشه و کف مجموعه به هیچ عنوان ضرر نکنه بنابر این جناب گوهر بین راس وینرز تیم با کمک جناب سعادت راس وینرز تیم در مشهد که هر دو خودشون طراح پلن هم هستن پلنی رو ریختن و به چند تا کمپانی که در زمینه رفرال مارکتینگ به صورت حرفه ای داشتند کار میکردن ارائه دادن و گفتن ما به عنوان زیر مجموعه ای مستقل از شما با نام تجاری خودمون و سایت خودمون لی با خدمات شما حاظر به همکاری هستیم که قوی ترینشون یعنی کمپانی ام وی که ۳ ساله به صورت حرفه در زمینه رفرال مارکتینگ در ایران فعالیت داره پذیرفت و ما یعنی کل وینرز تیم که ۸۰۰۰ نفر تقریبا بودیم شیفت شدیم داخل سایت خودمون و هیچ ربطی هم دیگه به بادران نداریم ولی خیلی قدرتمند تریم هم از نظر طرف قرارداد ها و هم از نظر سابقه در رفرال مارکتینگ

**Reply by saeedkousha** [reply]
توضیحات یکی از اعضای خودشون

**Reply by samira1370** [reply]
به مردم اطلاعات غلط نده اگه کلاهبرداری میخواست بکنه مجوز نمی‌گرفت اگه مجوز گرفته قبلا کلاهبرداری می‌کرد می‌گرفتنش دوباره مجوز نمیدادن در ضمن آقای گوهر بین یکی از هدلگاشه اصلا کاره ای نیست اونا ما درست کردیم در ضمن مثل شما بیکار نیستین منفی جایی رو بزنیم شما اگه به فکر مردم بودین به جای یه کار آفرینی خوب اینجا چرت نمیگفتین خودتون نمیتونین به مردم کمک کنین چرا به سیستم های خوب گیر میدین مکارم جرجندی و شخصیت های فیک دوروبرش کارشون شده این من نمیگم سیستم کلاهبرداری توی ایران نیست چرا هست ولی همه که کلاهبردار نیستن  اصلا ما میگیم باشه مثلا حرف شما قبول میشه بگی شما خودت چه گلی به سر مردم زدی و با اکانت های دیگت میای اینجا برای خودت حرف میزنی ؟ شما اگه مردشین یه سیستم هرچیزی میخواین خوب بیارین بالا معرفی کنین اگه نمیتونین بهتره بشینین نگاه کن و از پیشرفت کشور حسودی نکنین که خداوند حسودان را دوست ندارد و من الله توفیق...

**Reply by samira1370** [reply]
این حرف ها چیه میزنین دروغ نگین به ملت 5ساله داره کار میکنه من 2ساله دارم کار میکنم و الانم فعاله هرکسی هم میگه دروغ میگم میتونه با لینک سایت بره ببینه و استفاده کنه 

www.artanlife.com/Account/SignUp?refCode=60055

ای دی تلگرام خودمم اینجا میذارم که اینقد چرت نگین درضمن هدلگ سیستم هم توی مشهده و من توی این دوسالی که هستم خودشون میگن هرموقع مشکل داری بهمون زنگ بزن شماره هاشونم دارم آیدی تلگرام خودمو پایین این پیام میذارم اونایی که خواستن ببینن چیه درخدمتم

@BLOCK_TNIH

**Comment by saeedkousha** [comment]
این کمپانی ایرانی است و استارتش از مشهد زده شده
و الان در زاهدان و سبزه وار و... فعالیت دارد

**Reply by arshad511** [reply]
بله مشخص هستش که بیشتر کسب و کارهایی رو که در لیست دارند همه مغازه هایی در سطح شهر مشهد هستش.

**Reply by samira1370** [reply]
بهتره نگاه کنی شهرهای دیگم هست

**Comment by saeedkousha** [comment]
https://www.iranhoshdar.ir/Default.aspx?PageName=Newsb&ID=41499&Language=1

نظر ایران هشدار
البته اینجا گفته شبکه توزیع آرتان

**Comment by saeedkousha** [comment]
https://instagram.com/artan.life?utm_medium=copy_link
https://instagram.com/artanlife_discount?utm_medium=copy_link

**Reply by arshad511** [reply]
https://instagram.com/artan.life
https://instagram.com/artanlife_discount

لینک های پیج اینستاگرام

**Reply by samira1370** [reply]
این نیست دروغ نگین به ملت خودتون زدین اینو

**Comment by arshad511** [comment]
ظاهرن آقای رامین گوهربین از سرشبکه های این مجموعه کلاهبرداری هستش و مقامات قضایی لطفن از الان حواسشون له ایشون باشه

**Reply by samira1370** [reply]
من توی ارتان هستم و اصلا گوهر بین نمیشناسم چرا الکی میگی

**Comment by Arashparsian** [comment]
دوستان عزیز سلام 
این شبکه آرتان لایف نه تنها کلاهبرداری نیست بلکه درحال کمک به قشر وسیعی از جامعه در قالب حمایت از کسب و کار و پایین آوردن هزینه های خانوار می باشد
برای حمایت از خانواده های خودتون روی لینک زیر بزنید و از مزایای بی نظیر آرتان لایف بهره مند شوید

www.artanlife.com/Account/SignUp?refCode=37283

**Comment by Arashparsian** [comment]
خدارو شکر میکنم به این خانواده بزرک که روز به روز درحال رشد هست پیوستم و به کشور خودم  به مردم خودم و از کسب و کار های ملت عزیزم دارم حمایت و خدمت میکنم
شما هم همراه این راه باشید که خیر دنیا و آخرت رو به همراه داره

**Comment by samira1370** [comment]
ممنونم از پاسخ شما من هم در ارتان لایف در تیم بچه های مشهد و تهران ورود
کردم و هرکسی پلن نویس و اقتصاد دان باشد کاملا متوجه می‌شود که در این سایت
هیچگونه سرمایه گذاری انجام نمی‌شود و خرید و فروش واقعی صورت می‌گیرد توجیه
اقتصادی در ارتان لایف فوق تصور است
و کانال وب آموز با مدریت مکارم جرجندی ازین موضوع نا آگاه هستند و مردم
همین‌گونه که از کلاه برداران گول می‌خورند گول چنین اشخاصی هم خورده اند و
خودشان این موضوعات را پیگیری نمی‌کنند

من همچنان درحال رشد و توسعه هستم و دوستان می‌توانند در تلگرام آدرس کانال:
 @ArtanLife_Vip

را سرچ کنند و به گروه ما بپیوندند
